Add Content Template section to docs

mateoholman opened the pull request:
This PR will add a section for Content Templates to the Getting Started -> Back Office section of the docs, per Issue #989 
I'm open to any suggested edits / changes!

dampee commented on Getting-Started/Backoffice/Content-Templates/index.md:
```diff
@@ -0,0 +1,66 @@
+# Content Templates
+Content Templates allow you to create a blueprint for new content nodes based on an existing node.
```
Can we replace "you" with a role?  E.g. content editor?

mateoholman replied:
Hi @dampee I replaced this instance of you with the suggested 'content editor'. Did you want to replace all instances of 'you'?

dampee commented on Getting-Started/Backoffice/Content-Templates/index.md:
```diff
@@ -0,0 +1,66 @@
+
+Select the 'Content' menu: 
+
+![Content Menu](../images/01-Content-Menu.png)
```
Going one level up with images won't work on the OUR documentation website as that according to @jmayntzhusen .  We need to have images in a subfolder of the current file.

mateoholman replied:
No problem, done!

dampee commented on Getting-Started/Backoffice/Content-Templates/index.md:
```diff
@@ -0,0 +1,66 @@
+
+![New Content Template](../images/05-Find-Content-Template.png)
+
+## *Create - Method 2*
```
having bold and a header is a little overkill, right?  Just remove the star's

mateoholman commented:
@dampee I setup a profile on our.umbraco.com with the username mateoholman@gmail.com.
I think I addressed all your comments, but let me know if you see anything else!

dampee commented:
That's almost ace.  There was a small typo, but I fixed it already.  :-)

Thanks a lot!

nul800sebastiaan commented:
Hi there @mateoholman,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators
